src/modules/fetchRes.js

```diff
@@ -0,0 +1,17 @@
+// Get .env variables from webpack
+const { MYCLIENTID, MYCLIENTSECRET } = process.env;
```

Comment:
- [ ] Kindly give instructions on your README on how the API Key is accessed so one can be able to use it and access your App. 👍. Please note, do not push the API Keys to GitHub, you can only give instructions on how one can access it and use it 👍